modules/configure-web-terminal-image-admin.adoc

```diff
@@ -0,0 +1,21 @@
+// Module is included in the following assemblies:
+//
+// * web_console/web_terminal/configuring-web-terminal.adoc
+
+:_content-type: PROCEDURE
+
+[id="configure-web-terminal-image-admin_{context}"]
+= Configuring the web terminal image for all users
+
+You can use the *Administrator* perspective of the web console to set the default web terminal image for all users.
+
+.Prerequisites
+
+* You have cluster administrator permissions and are logged in to the web console.
+* You have installed the {web-terminal-op}.
+
+include::snippets/access-cluster-configuration-console.adoc[]
+
+. Click the *Web Terminal* tab, which opens the *Web Terminal Configuration* page.
+. Enter the URL of the image that you want to use.
+. Click the *Save* button to save your changes.
```

Comment:
```suggestion
. Click *Save*.
```
- Per ISG, "In most cases, refer to a button by its label only": https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#button.
- Consider removing `to save your changes` for the sake of minimalism; to me, it's redundant and states the obvious.